data/beaconrestapi/src/test/java/tech/pegasys/artemis/beaconrestapi/beaconhandlers/BeaconValidatorsHandlerTest.java

```diff
@@ -292,17 +289,16 @@ public void shouldReturnBadRequestWhenBadEpochParameterSpecified() throws Except
 
   @Test
   public void shouldReturnEmptyListWhenFutureEpochSpecified() throws Exception {
```

Comment:
this sort of functional test i think is testing the provider, so i think it should probably live there, so that if we write a new interface we don't have to duplicate this test...